deploy/cloudformation/elastic-agent-ec2.yml

```diff
@@ -156,6 +156,10 @@ Resources:
           tar xzvf $ElasticAgentArtifact.tar.gz
           cd $ElasticAgentArtifact
           sudo ./elastic-agent install --non-interactive --url=${FleetUrl} --enrollment-token=${EnrollmentToken}
+      BlockDeviceMappings:
+        - DeviceName: /dev/xvda
+          Ebs:
+            VolumeSize: 10
```

Comment:
got "not enough space on disk" when downloading elastic-agent without it. (default for this AMI looks like 2gb)

Reply:
could be related? https://github.com/elastic/cloudbeat/issues/1065

Reply:
yeah seems like it. i got that error after re-downloading the agent, which is about 2g when unpacked. i assume an upgrade flow does the same

Reply:
I would change it to 16 or even 32 just to be on the safe side.